spec/index.html

```diff
@@ -370,11 +370,11 @@ <h3>Terms defined by this specification</h3>
       <dt><dfn>hash</dfn></dt>
       <dd>The lowercase, hexadecimal representation of a message digest.</dd>
       <dt><dfn>hash algorithm</dfn></dt>
-      <dd>The hash algorithm used by <a>RDFC-1.0</a>, namely, SHA-256.
-        <div class="note">Implementations can be written to parameterize the
-        hash algorithm without any other changes. However, using a different
-        hash algorithm is expected to generate different output from
-        <a>RDFC-1.0</a>.</div>
+      <dd>The default hash algorithm used by <a>RDFC-1.0</a>, namely, SHA-256.
+        <p>Implementations can be written to parameterize the
+          hash algorithm without any other changes. Using a different
+          hash algorithm is expected to generate different output from
+          using the default.</p>
```

Comment:
I think here and in the Insecure Hash Algorithm section it needs to be said in stronger words that the canonical output depends on the hash algorithm used.

Reply:
"is expected to" => "will generally" (it might produce the same output). What more would you like this to say?

Reply:
I suppose it's fine.  I wanted to make sure no one got tripped up on the issue that this is an important input parameter.